src/client.rs

```diff
@@ -233,10 +243,12 @@ fn init_title_screen(s: &mut MainState) -> Result<(), ()> {
     // 3) Center it
     // 4) get offset for row and column to draw at
 
-//  let resolution = s.video_settings.get_active_resolution();
+    // let resolution = s.video_settings.get_active_resolution();
     let resolution = (config::DEFAULT_SCREEN_WIDTH, config::DEFAULT_SCREEN_HEIGHT);
-    let win_width = resolution.0 as i32 / s.viewport.get_cell_size() as i32;
-    let win_height = resolution.1  as i32 / s.viewport.get_cell_size() as i32;
+    let win_width  = resolution.0 as i32 / s.viewport.get_cell_size() as i32;
```

Comment:
I think we ran into an issue with using the saved `cell_size`, so we had to modify `Conwayste.toml` to get around the issue. I think it would be best if we use the default cell size here instead of getting it from the viewport.

Reply:
We discussed putting constants in their own module which other modules can reference. Thanks for making this change.
`constants.rs`? `consts.rs`?
`definitions.rs`? `defs.rs`?